automatic/sourcetree/tools/chocolateyInstall.ps1

```diff
@@ -1,17 +1,17 @@
 ﻿$ErrorActionPreference = 'Stop'
 
 $packageArgs = @{
-  packageName = 'SourceTree'
-  fileType = 'EXE'
-  url = 'https://downloads.atlassian.com/software/sourcetree/windows/ga/SourceTreeSetup-2.5.5.exe'
-
-  softwareName = 'SourceTree'
-
-  checksum = 'b4309a4581a515d3498a75e9c810ae2b16adf239107486f481d9a0236240ee5f'
-  checksumType = 'sha256'
-
-  silentArgs = "/passive"
-  validExitCodes = @(0, 3010, 1641)
+  packageName   = $env:ChocolateyPackageName
+  softwareName  = 'Sourcetree*'
+  fileType      = 'msi'
+  silentArgs    = "/qn /norestart ACCEPTEULA=1 /l*v `"$env:TEMP\$env:ChocolateyPackageName.$env:ChocolateyPackageVersion.log`""
+  validExitCodes= @(0,1641,3010)
+  url           = 'https://downloads.atlassian.com/software/sourcetree/windows/ga/SourcetreeEnterpriseSetup_2.5.5.msi'
+  checksum      = '2fb9c8676259f8d74acd50e6069bc93c2ca7c3e9305341fa6e0571df0d7d74e9'
+  checksumType  = 'sha256'
+  url64bit      = ''
+  checksum64    = ''
+  checksumType64= 'sha256'
 }
 
 Install-ChocolateyPackage @packageArgs
```

Comment:
what about updates, is the new MSI installer subjected to the same kind of error during update as quite a few MSIs have?